### ISSUE TITLE
train loss do not down

### ISSUE THREAD

**Issue by NileZhou** [NVIDIA/tacotron2#230]
I train it in a GTX1080Ti, set batch_size as 16, other params is default. use ljspeech dataset
now it is run at 2k+ steps。the train loss always Oscillation around 0.4, grad norm Oscillation around 1.0. 
These two figures have not declined for a long time.

**Comment by rafaelvalle** [comment]
Try annealing the learning rate.
